cmd/kubeadm/OWNERS

```diff
@@ -1,18 +1,18 @@
 approvers:
-- jbeda
 - luxas
-- krousey
 - timothysc
+- fabriziopandini
 reviewers:
 - luxas
-- dmmcquay
-- krousey
 - timothysc
 - fabriziopandini
-- jamiehannaford
 - kad
 - xiangpengzhao
-- kargakis
+- stealthybox
+- liztio
+- chuckha
+- detiber
+// Might want to add @kargakis, @jamiehannaford, @krousey and/or @dmmcquay again
```

Comment:
@luxas I think this needs to be a yaml comment (starts with `#` not `//`)

Reply:
Oh true this is a yaml file. Will do

Reply:
@luxas I'd like to join in. Could you pls add me as well.

Reply:
Yes, in the SIG meeting I recalled you already were on this list and hence didn't mention adding you :joy:, but yes, you should of course be here given the many reviews you've done and successfully participating in the pilot mentoring programme!